server/src/tables.rs

```diff
@@ -21,6 +21,8 @@ use std::task::{Context, Poll};
 
 use async_trait::async_trait;
 
+use tracing::{debug, error, info, warn, Level, event, instrument, span};
```

Comment:
Imports should be grouped as in `catalog.rs`, `main.rs`, and `remote.rs`.

Reply:
Fixed

Reply:
I am not sure this is fixed.

Reply:
You are absolutely right. Now it should be fixed.